### PR TITLE
Add entry page facet and update member id type enum and convert xml b…

### PR DESCRIPTION
**Pull request by LeonardoGonzales**
Add entry page facet and update member id type enum and convert xml behaviour

### CODE REVIEW

File: spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniref/converter/DatasetUniRefEntryLightConverter.java [hunk 1]
```diff
@@ -111,9 +111,11 @@ private void convertCommonProperties(Row rowValue, UniRefEntryLightBuilder build
 
     private void addMemberInfo(UniRefEntryLightBuilder builder, UniRefMember member) {
         // member accessions
+        int memberType = member.getMemberIdType().getDisplayOrder();
         member.getUniProtAccessions().stream()
```
Comment (ahmadshadab) [inline spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniref/converter/DatasetUniRefEntryLightConverter.java:115]: the variable name "memberType"  should be related to order.

File: spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniref/converter/DatasetUniRefEntryLightConverter.java [hunk 2]
```diff
@@ -111,9 +111,11 @@ private void convertCommonProperties(Row rowValue, UniRefEntryLightBuilder build
...
                 .map(Value::getValue)
                 .findFirst()
+                .map(id -> id + "," + memberType)
                 .ifPresent(builder::membersAdd);
```
Comment (ahmadshadab) [inline spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniref/converter/DatasetUniRefEntryLightConverter.java:119]: Lets discuss about this comma logic.

Comment (LeonardoGonzales) [reply spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniref/converter/DatasetUniRefEntryLightConverter.java:119]: commented the code as well

File: spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniref/converter/DatasetUniRefEntryLightConverter.java (hunk 1 above)
Comment (LeonardoGonzales) [reply spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniref/converter/DatasetUniRefEntryLightConverter.java:115]: I think if we put order in the name will make it more confuse... what name do you suggest?

**Pull request merged**